src/components/WorkersLateIndicator.vue

```diff
@@ -16,20 +16,41 @@
   import { useSubscription } from '@prefecthq/vue-compositions'
   import { computed, toRefs } from 'vue'
   import { useWorkspaceApi } from '@/compositions'
+  import { UnionFilters } from '@/types'
   import { toPluralString } from '@/utilities'
 
   const props = defineProps<{
     workPoolName: string,
     workPoolQueueNames?: string[],
   }>()
 
-  const workPoolQueueNames = computed(() => props.workPoolQueueNames ?? [])
 
   const api = useWorkspaceApi()
+
   const { workPoolName } = toRefs(props)
+  const workPoolQueueNames = computed(() => props.workPoolQueueNames ?? [])
+
+
+  const flowRunFilter = computed<UnionFilters>(() => {
+
+    const flowRunFilter: UnionFilters = {
+      'work_pools': { name: { any_: [workPoolName.value] } },
+      'flow_runs': {
+        state: { name: { any_: ['Late'] } },
+      },
+      sort: 'START_TIME_ASC',
```

Comment:
can't wait for @pleek91 to give us a better syntax here 😀

Reply:
Me too! 😂

Reply:
It's future Craig's responsibilities

Reply:
that guy